integration-tests/run-scenario.sh

```diff
@@ -0,0 +1,178 @@
+  echo "Wait port: ${1}:${2}"
+  while ! nc -z $host $port; do
+    sleep 2 # wait for 2 second before check again
+    echo -n "."
+    if [ $x = $retry_count ]; then
+      echo "port never opened."
+      exit 1
+    fi
+  x=$((x+1))
+  done
+}
+
+#----------------------------------------------------------------------
+# select default username and remote directory based on the OS
+#----------------------------------------------------------------------
+case "${os}" in
+   "CentOS")
+    	user=centos
+        PROP_REMOTE_DIR=REMOTE_WORKSPACE_DIR_UNIX ;;
+   "Windows")
+    	user=Administrator
+        PROP_REMOTE_DIR=REMOTE_WORKSPACE_DIR_WINDOWS ;;
+   "UBUNTU")
+        user=ubuntu
+        PROP_REMOTE_DIR=REMOTE_WORKSPACE_DIR_UNIX ;;
+esac
+
+REM_DIR=`grep -w "$PROP_REMOTE_DIR" ${FILE1} ${FILE2} | cut -d'=' -f2`
+
+#----------------------------------------------------------------------
+# wait till port 22 is opened for SSH
+#----------------------------------------------------------------------
+wait_for_port ${host} 22
+
+#----------------------------------------------------------------------
+# execute commands based on the OS of the instance
+# Steps followed;
+# 1. SSH and make the directory.
+# 2. Copy necessary files to the instance.
+# 3. Execute scripts at the instance.
+# 4. Retrieve reports from the instance.
+#----------------------------------------------------------------------
+if [ "${os}" = "Windows" ]; then
+  echo "Waiting 4 minutes till Windows instance is configured. "
+  sleep 4m #wait 4 minutes till Windows instance is configured and able to receive password using key file.
+  set +o xtrace #avoid printing sensitive data in the next commands
+  request_ec2_password $instance_id
+  REM_DIR=$(echo "$REM_DIR" | sed 's/\\//g')
+  echo "Copying files to ${REM_DIR}.."
+  sshpass -p "${password}" scp -q -o StrictHostKeyChecking=no ${FILE1} ${user}@${host}:${REM_DIR}
+  sshpass -p "${password}" scp -q -o StrictHostKeyChecking=no ${FILE2} ${user}@${host}:${REM_DIR}
+  sshpass -p "${password}" scp -q -o StrictHostKeyChecking=no ${FILE3} ${user}@${host}:${REM_DIR}
+  sshpass -p "${password}" scp -q -o StrictHostKeyChecking=no ${FILE4} ${user}@${host}:${REM_DIR}
+  sshpass -p "${password}" scp -q -o StrictHostKeyChecking=no ${FILE5} ${user}@${host}:${REM_DIR}
+  sshpass -p "${password}" scp -q -o StrictHostKeyChecking=no ${FILE6} ${user}@${host}:${REM_DIR}
+  sshpass -p "${password}" scp -q -o StrictHostKeyChecking=no ${FILE8} ${user}@${host}:${REM_DIR}
+  
+  echo "=== Files copied successfully ==="
+  echo "Execution begins.. "
+
+  sshpass -p "${password}" ssh -o StrictHostKeyChecking=no ${user}@${host} "${REM_DIR}/${FILE8}" ${REM_DIR}
+  echo "=== End of execution ==="
+  echo "Retrieving reports from instance.. "
+  sshpass -p "${password}" scp -r -q -o StrictHostKeyChecking=no ${user}@${host}:${REM_DIR}/ballerinatestrepo/ballerina-lang/tests/ballerina-integration-test/target/surefire-reports ${DIR}  
```

Comment:
surefire reports path seems to be wrong. `ballerinatestrepo/` has to be removed as I see